python/tskit/trees.py

```diff
@@ -357,6 +357,19 @@ def __eq__(self, other):
             and self.metadata == other.metadata
         )
 
+    def alleles(self):
```

Comment:
Should this be a function or an attribute? The only reason I can think of for making it a function is if we can imagine adding some parameters later.

Reply:
Happy with either. I then to think of attributes as "low cost", which I guess this is.